{
  "cve": "CVE-2021-44647",
  "mitre": {
    "cpes": [],
    "created": "2022-01-11T00:00:00+00:00",
    "description": "Lua v5.4.3 and above are affected by SEGV by type confusion in funcnamefromcode function in ldebug.c which can cause a local denial of service.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2021/44xxx/CVE-2021-44647.json",
    "references": [
      "http://lua-users.org/lists/lua-l/2021-11/msg00195.html",
      "http://lua-users.org/lists/lua-l/2021-11/msg00204.html",
      "https://access.redhat.com/security/cve/cve-2021-44647",
      "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/P3EMGAQ5Y6GXJLY4K5DUOOEQT4MZ4J4F/",
      "https://security.gentoo.org/glsa/202305-23"
    ],
    "title": null,
    "updated": "2024-08-04T04:25:16.894000+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:lua:lua:5.4.3:*:*:*:*:*:*:*",
      "cpe:2.3:o:fedoraproject:fedora:34:*:*:*:*:*:*:*"
    ],
    "created": "2022-01-11T13:15:07.833000+00:00",
    "description": "Lua v5.4.3 and above are affected by SEGV by type confusion in funcnamefromcode function in ldebug.c which can cause a local denial of service.",
    "metrics": {
      "cvssV2_0": {
        "score": 2.1,
        "vector": "AV:L/AC:L/Au:N/C:N/I:N/A:P"
      },
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 5.5,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
      },
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2021/CVE-2021-44647.json",
    "references": [
      "http://lua-users.org/lists/lua-l/2021-11/msg00195.html",
      "http://lua-users.org/lists/lua-l/2021-11/msg00204.html",
      "https://access.redhat.com/security/cve/cve-2021-44647",
      "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/P3EMGAQ5Y6GXJLY4K5DUOOEQT4MZ4J4F/",
      "https://security.gentoo.org/glsa/202305-23"
    ],
    "title": null,
    "updated": "2023-11-07T03:39:40.740000+00:00",
    "vendors": [
      "fedoraproject",
      "fedoraproject$PRODUCT$fedora",
      "lua",
      "lua$PRODUCT$lua"
    ],
    "weaknesses": [
      "CWE-843"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:2.3:a:lua:lua:5.4.3:*:*:*:*:*:*:*",
        "cpe:2.3:o:fedoraproject:fedora:34:*:*:*:*:*:*:*"
      ],
      "providers": [
        "nvd"
      ]
    },
    "created": {
      "data": "2022-01-11T00:00:00+00:00",
      "provider": "mitre"
    },
    "description": {
      "data": "Lua v5.4.3 and above are affected by SEGV by type confusion in funcnamefromcode function in ldebug.c which can cause a local denial of service.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {
          "score": 2.1,
          "vector": "AV:L/AC:L/Au:N/C:N/I:N/A:P"
        },
        "provider": "nvd"
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {
          "score": 5.5,
          "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
        },
        "provider": "nvd"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": "Low",
        "provider": "redhat"
      }
    },
    "references": {
      "data": [
        "http://lua-users.org/lists/lua-l/2021-11/msg00195.html",
        "http://lua-users.org/lists/lua-l/2021-11/msg00204.html",
        "https://access.redhat.com/security/cve/cve-2021-44647",
        "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/P3EMGAQ5Y6GXJLY4K5DUOOEQT4MZ4J4F/",
        "https://nvd.nist.gov/vuln/detail/CVE-2021-44647",
        "https://security.gentoo.org/glsa/202305-23",
        "https://www.cve.org/CVERecord?id=CVE-2021-44647"
      ],
      "providers": [
        "mitre",
        "nvd",
        "redhat"
      ]
    },
    "title": {
      "data": "lua: type confusion in funcnamefromcode in ldebug.c could result in local DoS",
      "provider": "nvd"
    },
    "updated": {
      "data": "2024-08-04T04:25:16.894000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "fedoraproject",
        "fedoraproject$PRODUCT$fedora",
        "lua",
        "lua$PRODUCT$lua"
      ],
      "providers": [
        "nvd"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-843"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    }
  },
  "redhat": {
    "cpes": [],
    "created": "2022-01-11T00:00:00+00:00",
    "description": "Lua v5.4.3 and above are affected by SEGV by type confusion in funcnamefromcode function in ldebug.c which can cause a local denial of service.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 5.5,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
      },
      "threat_severity": "Low"
    },
    "redhat_repo_path": "2021/CVE-2021-44647.json",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2021-44647",
      "https://www.cve.org/CVERecord?id=CVE-2021-44647"
    ],
    "title": "lua: type confusion in funcnamefromcode in ldebug.c could result in local DoS",
    "updated": "2022-01-11T00:00:00+00:00",
    "vendors": [],
    "weaknesses": [
      "CWE-843"
    ]
  }
}